platform/features/plot/src/services/ExportImageService.js

```diff
@@ -76,17 +76,21 @@ define(
             }, self.EXPORT_IMAGE_TIMEOUT);
 
             try {
+                element.style.backgroundColor = 'white';
+                
                 self.html2canvas(element, {
                     onrendered: function (canvas) {
                         switch (type.toLowerCase()) {
                             case "png":
                                 canvas.toBlob(defer.resolve, "image/png");
+                                element.style.backgroundColor = '';
```

Comment:
You should store the original background color in a variable and reset it afterwards.  otherwise you might overwrite an existing style.